src/uilib-test-renderer/helper.ts

```diff
@@ -12,4 +13,8 @@ const findStyle = <T>(key: string, component: Component): T => {
   return StyleSheet.flatten(component.props.style)[key];
 };
 
-export {findStyle};
+const FireOnMomentumScrollEnd = (component: any, {x, y}: {x?: number, y?: number}) => {
```

Comment:
You need to use lower case for functions.
Also, 'helper' is a very general name. I would keep this file for scrollView helper functions.
The 'nativeEvent' configuration I would move to a separate function, as you might add more helper functions that will need it, like 'onScroll'.